app-admin/logcheck/files/logcheck.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Mails anomalies in the system logfiles to the administrator
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/nice -n10 /usr/sbin/logcheck
```

Comment:
nice in systemd ExecStart is not idiomatic. One would use CpuWeight and Co. for this purpose. But I lean towards this being a policy decision of the administrator in the case of logcheck. And admins can use systemd's override mechanism to configure the service to their needs.